common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -34,9 +34,9 @@ def _get_element_text(self, selector):
             return None
 
 
-class EdxNotesPageGroup(NoteChild):
+class EdxNotesChapterGroup(NoteChild):
     """
-    Helper class that works with note groups on Note page of the course.
+    Helper class that works with chapter (section) grouping of notes in the Course Structure view on the Note page.
```

Comment:
I believe that docstrings are kept to 79 characters wide, even though code can go to 120.